src/components/chips/chips.scss

```diff
@@ -0,0 +1,51 @@
+$chip-font-size: 13px;
```

Comment:
Add !default to each of these so that someone could theoretically override them.